internal/job/starter.go

```diff
@@ -4,18 +4,19 @@ import "context"
 
 // StartOptions represents the options for starting a job in the Sauce Labs cloud.
 type StartOptions struct {
-	User           string        `json:"username"`
-	AccessKey      string        `json:"accessKey"`
-	App            string        `json:"app,omitempty"`
-	Suite          string        `json:"suite,omitempty"`
-	Framework      string        `json:"framework,omitempty"`
-	BrowserName    string        `json:"browserName,omitempty"`
-	BrowserVersion string        `json:"browserVersion,omitempty"`
-	PlatformName   string        `json:"platformName,omitempty"`
-	Name           string        `json:"name,omitempty"`
-	Build          string        `json:"build,omitempty"`
-	Tags           []string      `json:"tags,omitempty"`
-	Tunnel         TunnelOptions `json:"tunnel,omitempty"`
+	User             string        `json:"username"`
+	AccessKey        string        `json:"accessKey"`
+	App              string        `json:"app,omitempty"`
+	Suite            string        `json:"suite,omitempty"`
+	Framework        string        `json:"framework,omitempty"`
+	FrameworkVersion string        `json:"frameworkVersion,omitempty"`
```

Comment:
This might work for Cypress but is kind of ambiguous for e.g. Puppeteer or Playwright where this seems to describe the automation tool but could also mean the version of Jest. How is `FrameworkVersion` determined for these cases?

Reply:
This `frameworkVersion` is expected to contains the actual version of the framework.
As a first step we will only support it for framework. We may add support for the automation tool version later if there is demand. 

May be the template `config.yml` file should be updated to remove this ambiguity part (add comment near this settings value).

Reply:
It should be at least clear which for which framework this field has support.

Reply:
This will be effective for every framework, this is closely related to "native config" that we have implemented.
I will add more details about this to ensure we know which version is currently supported by this setting.